src/includes/capture-error/javascript.mdx

```diff
@@ -1,3 +1,7 @@
+By including and configuring Sentry, our JavaScript SDK automatically attaches global handlers to _capture_ uncaught exceptions and unhandled promise rejections, as described in the official ECMAScript 6 standard. You can disable this default behavior by changing the `onunhandledrejection` option to `false` in your GlobalHandlers integration and manually hook into each event handler, then call `Sentry.captureException` or `Sentry.captureMessage` directly.
```

Comment:
aside we might want a diff template for this - as i think capture-error is used under Usage right? i assume this is added because its used somewhere else in addition

Reply:
I may have moved it to the wrong place - I was revising work from an old branch. Will fix. Thanks.

Reply:
Moved this content into troubleshooting.